styx-service-common/src/main/java/com/spotify/styx/storage/InMemStorage.java

```diff
@@ -61,18 +60,8 @@
   private final ConcurrentMap<WorkflowId, WorkflowState> workflowStatePerWorkflowId = Maps
       .newConcurrentMap();
 
-  public final List<SequenceEvent> writtenEvents = Lists.newCopyOnWriteArrayList();
-  public final Map<WorkflowInstance, RunState> activeStatesMap = Maps.newHashMap();
-
-  public final CountDownLatch countDown;
-
-  public InMemStorage() {
-    this(0);
-  }
-
-  public InMemStorage(int expectedWorkflowExecutionInfoStored) {
-    this.countDown = new CountDownLatch(expectedWorkflowExecutionInfoStored);
-  }
+  private final List<SequenceEvent> writtenEvents = Lists.newCopyOnWriteArrayList();
```

Comment:
Is the `InMemStorage` class used anywhere? I cannot find where is used in the code base.

Reply:
This is legacy and I agree we can remove it. Let's do it in a separated PR.